---
permalink: /reference/2023-GLPRD/
collection: reference
title: "2023 Great Lakes Pediatric Research Day"
author_profile: true
---
# Microbiome and metabolome dynamics during the first year of life
**Douglas V. Guzior<sup>1,2</sup>**, Hao Wu<sup>3</sup>, Christian Martin<sup>2</sup>, Madison R. Rzepka<sup>2</sup>, Kerri A. Neugebauer<sup>4</sup>, Julie Lumeng<sup>5-7</sup>, Gustavo de los Campos<sup>3,8</sup>, Robert A. Quinn<sup>2</sup>

1. Department of Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI
2. Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI
3. Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, MI
4. Department of Plant Soil and Microbiology, Michigan State University, East Lansing, MI
5. Center for Human Growth and Development, University of Michigan Medical School, Ann Arbor, MI
6. Department of Pediatrics, University of Michigan Medical School, Ann Arbor, MI
7. Department of Nutritional Sciences, University of Michigan School of Public Health, Ann Arbor, MI
8. Department of Statistics and Probability, Michigan State University, East Lansing, MI

## Abstract
Early life exposures of the neonate can affect long term health outcomes, particularly in the context of childhood asthma, obesity, and diabetes<sup>1</sup>. While early life microbial exposures have been well characterized in recent years showing important inoculation events related to birth route (i.e., natural or cesarian section)<sup>1,2</sup>, little is known about changes in gut metabolites at this crucial developmental stage. Here, we characterize infant fecal microbiome and metabolome development across the first 12 months of life for 233 mother-child pairs. We found that microbiome Shannon diversity increases as children get older, with marked abundance increases in amplicon sequence variants (ASVs) associated with *Romboutsia*, *Ruminococcus*, *Faecalibacterium*, and *Blautia*. Metabolome shifts over time were analyzed via untargeted mass spectrometry analysis of thousands of molecular features detected in feces. In contrast to the microbiome, metabolite Shannon diversity significantly decreased as a function of time, driven by reduced metabolite richness. This counter-correlation may be the consequence of microbial niche-filling, resulting in further metabolism of previously unusable metabolites.  Random Forest regression analysis of fecal metabolomes found that infant age explained 68% of metabolome variance across samples, showing that the infant fecal metabolome undergoes maturation in much the same way as the fecal microbiome. SIRIUS<sup>10</sup> spectral analysis of the top contributors to this model revealed that 16 of the top 30 contributors to this model are predicted steroid compounds, such as bile acids (BAs). Our lab is focused on microbial metabolism of human BAs, particularly that of BA reconjugation. We found that the proportion of samples containing microbially-conjugated bile acids (MCBAs) decreased with time. Little is known about the role MCBAs play in the infant gut. MCBAs are primarily associated with fecal metabolomes of patients experiencing various disease states, such as IBD and Crohn's disease, warranting further investigation into their role in human disease and development. One avenue is fat absorption, where MCBAs may play a significant role in nutrient acquisition while the infant is primarily fed through mother’s milk, with high concentrations during gut dysbiosis acting as a microbiota-driven method of compensating for nutrient malabsorption.

## Poster References
1. Stewart, C. J. et al. Temporal development of the gut microbiome in early childhood from the TEDDY study. *Nature* **562**, 583–588 (2018).

2. Yao, Y. et al. The Role of Microbiota in Infant Health: From Early Life to Adulthood. *Front. Immunol.* **12**, 708472 (2021).

3. Wahlström, A., Sayin, S. I., Marschall, H.-U. & Bäckhed, F. Intestinal Crosstalk between Bile Acids and Microbiota and Its Impact on Host Metabolism. *Cell Metab.* **24**, 41–50 (2016).

4. Guzior, D. V. & Quinn, R. A. Review: microbial transformations of human bile acids. *Microbiome* **9**, 140(2021).

5. Quinn, R. A. et al. Global chemical effects of the microbiome include new bile-acid conjugations. *Nature* **579**, 123–129 (2020).

6. Lucas, L. N. et al. Dominant bacterial phyla from the human gut show widespread ability to transform and conjugate bile acids. *mSystems* **6**, e00805-21 (2021).

7. Pluskal, T., Castillo, S., Villar-Briones, A. & Orešič, M. MZmine 2: Modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data. *BMC Bioinform.* **11**, 395 (2010).

8. Myers, O. D., Sumner, S. J., Li, S., Barnes, S. & Du, X. One step forward for reducing false positive and false negative compound identifications from mass spectrometry metabolomics data: New algorithms for constructing extracted ion chromatograms and detecting chromatographic peaks. *Anal. Chem.* **89**, 8696–8703 (2017).

9. Nothias, L.-F. et al. Feature-based molecular networking in the GNPS analysis environment. *Nat. Methods* **17**, 905–908 (2020).

10. Dührkop, K. et al. SIRIUS 4: a rapid tool for turning tandem mass spectra into metabolite structure information. *Nat. Methods* **16**, 299–302 (2019).

11. Gonzalez, A. et al. Qiita: rapid, web-enabled microbiome meta-analysis. *Nat. Methods* **15**, 796–798 (2018).

12. Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. *Nat. Biotech.* **37**, 852–857 (2019).

13. Yilmaz, P. et al. The SILVA and ‘all-species Living Tree Project (LTP)’ taxonomic frameworks. *Nucleic Acids Res.* **42**, D643-8 (2014).

14. Quast, C. et al. The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools. *Nucleic Acids Res* *41*, D590-6 (2013).

15. R Core Team. R: A Language and Environment for Statistical Computing. (2022).

16. Oksanen, J. et al. vegan: Community Ecology Package. (2022).

17. Yamato, Y., Kimura, A., Inoue, T., Kurosawa, T. & Kato, H. Fetal Bile Acid Metabolism: Analysis of Urinary 3β-Monohydroxy-Δ5 Bile Acid in Preterm Infants. *Neonatology* **80**, 19–25 (2001).

18. Kimura, A. et al. Fetal bile acid metabolism during infancy: Analysis of 1β-hydroxylated bile acids in urine, meconium and feces. *Hepatology* **20**, 819–824 (1994).

19. Lopez-Siles, M., Duncan, S. H., Garcia-Gil, L. J. & Martinez-Medina, M. *Faecalibacterium prausnitzii*: from microbiology to diagnostics and prognostics. *ISME J* **11**, 841–852 (2017).
