---
title: "Dissertation Defense"
collection: reference
permalink: /reference/2024-Defense/
author_profile: true
date: 2024-04-24
---
# Catalytic mechanisms and physiological consequences of microbial bile acid conjugation
**Douglas V. Guzior<sup>1,2</sup>**

1. Department of Microbiology, Genetics, and Immunology, Michigan State University, East Lansing, MI, USA
2. Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI, USA

## Abstract
Bile acids (BAs) are an integral component of the human digestion and gastrointestinal health. They function as detergents for nutrient absorption, antimicrobial compounds, and signaling molecules. It has long been known that humans produce both glycine and taurine conjugated primary BAs in the liver from cholesterol, storing them in the gallbladder prior to a meal. Production of these conjugated BAs was thought to be performed exclusively by the host for decades until the gut bacterium <i>Enterocloster bolteae</i> was shown to produce phenylalanine, tyrosine, and leucine conjugated cholate, the first described microbially conjugated bile acids (MCBAs). However, little was known about how MCBAs are made, which microbes produce them, and least of all, their impacts on human health. In this seminar, I will describe the discovery that acyl transfer activity from the enzyme bile salt hydrolase (BSH), one of the most ubiquitous enzymes produced across gut microbiome, is responsible for the production of MCBAs. Diverse BSH enzymes from diverse gut microbes can produce a plethora of MCBAs, greatly expanding the chemistry of the human bile acid pool. Leveraging murine models, I will describe how MCBAs impact the host microbiome and can enter circulation throughout the body. Finally, I show how MCBA concentrations change within both the dysbiotic gut, through a sleeve gastrectomy patient cohort, and within the developing gut, through an infant cohort. Taken together, this work teases apart the “who” and “how” of microbial bile acid conjugation, while also contributing to our understanding of the grand question of “why”.

## References

Coleman JP, Hudson LL. 1995. Cloning and characterization of a conjugated bile acid hydrolase gene from <i>Clostridium perfringens</i>. Appl Environ Microbiol 61:2514–2520. doi: [10.1128/aem.61.7.2514-2520.1995](doi.org/10.1128/aem.61.7.2514-2520.1995)

de Aguiar Vallim TQ, Tarling EJ, Edwards PA. 2013. Pleiotropic roles of bile acids in metabolism. Cell Metab 17:657–669. doi: [10.1016/j.cmet.2013.03.013](doi.org/10.1016/j.cmet.2013.03.013)

Ding L, Zhang E, Yang Q, Jin L, Sousa KM, Dong B, Wang Y, Tu J, Ma X, Tian J, Zhang H, Fang Z, Guan A, Zhang Y, Wang Z, Moore DD, Yang L, Huang W. 2021. Vertical sleeve gastrectomy confers metabolic improvements by reducing intestinal bile acids and lipid absorption in mice. Proc Natl Acad Sci USA 118:e2019388118. doi: [10.1073/pnas.2019388118](doi.org/10.1073/pnas.2019388118)

Foley MH, Walker ME, Stewart AK, O’Flaherty S, Gentry EC, Patel S, Beaty VV, Allen G, Pan M, Simpson JB, Perkins C, Vanhoy ME, Dougherty MK, McGill SK, Gulati AS, Dorrestein PC, Baker ES, Redinbo MR, Barrangou R, Theriot CM. 2023. Bile salt hydrolases shape the bile acid landscape and restrict <i>Clostridioides difficile</i> growth in the murine gut. Nat Microbiol 8:611–628. doi: [10.1038/s41564-023-01337-7](doi.org/10.1038/s41564-023-01337-7)

Folz J, Culver RN, Morales JM, Grembi J, Triadafilopoulos G, Relman DA, Huang KC, Shalon D, Fiehn O. 2023. Human metabolome variation along the upper intestinal tract. Nat Metab 5:777–788. doi: [10.1038/s42255-023-00777-z](doi.org/10.1038/s42255-023-00777-z)

Fu T, Huan T, Rahman G, Zhi H, Xu Z, Oh TG, Guo J, Coulter S, Tripathi A, Martino C, McCarville JL, Zhu Q, Cayabyab F, Low B, He M, Xing S, Vargas F, Yu RT, Atkins A, Liddle C, Ayres J, Raffatellu M, Dorrestein PC, Downes M, Knight R, Evans RM. 2023. Paired microbiome and metabolome analyses associate bile acid changes with colorectal cancer progression. Cell Rep 42:112997. doi: [10.1016/j.celrep.2023.112997](doi.org/10.1016/j.celrep.2023.112997)

Georgiou K, Belev NA, Koutouratsas T, Katifelis H, Gazouli M. 2022. Gut microbiome: Linking together obesity, bariatric surgery and associated clinical outcomes under a single focus. World J Gastrointest Pathophysiol 13:59–72. doi: [10.4291/wjgp.v13.i3.59](doi.org/10.4291/wjgp.v13.i3.59)

Gopal-Srivastava R, Hylemon PB. 1988. Purification and characterization of bile salt hydrolase from <i>Clostridium perfringens</i>. J Lipid Res 29:1079–1085. doi: [10.1016/s0022-2275(20)38464-9](doi.org/10.1016/s0022-2275(20)38464-9)

Guzior DV, Okros M, Shivel M, Armwald B, Bridges C, Fu Y, Martin C, Schilmiller AL, Miller WM, Ziegler KM, Sims MD, Maddens ME, Graham SF, Hausinger RP, Quinn RA. 2024. Bile salt hydrolase acyltransferase activity expands bile acid diversity. Nature 626:852–858. doi: [10.1038/s41586-024-07017-8](doi.org/10.1038/s41586-024-07017-8)

Guzior DV, Quinn RA. 2021. Review: microbial transformations of human bile acids. Microbiome 9(1):140. doi: [10.1186/s40168-021-01101-1](doi.org/10.1186/s40168-021-01101-1)

Hofmann AF. 1999. The continuing importance of bile acids in liver and intestinal disease. Arch Intern Med 159:2647–2658. doi: [10.1001/archinte.159.22.2647](doi.org/10.1001/archinte.159.22.2647)

Huang X, Weng P, Zhang H, Lu Y. 2014. Remodeling intestinal flora with sleeve gastrectomy in diabetic rats. Journal of Diabetes Research 2014:1–5. doi: [10.1155/2014/196312](doi.org/10.1155/2014/196312)

Karlov DS, Long SL, Zeng X, Xu F, Lal K, Cao L, Hayoun K, Lin J, Joyce SA, Tikhonova IG. 2023. Structure of <i>Lactobacillus salivarius</i> (Ls) bile salt hydrolase(BSH) in complex with taurocholate (TCA). Retrieved 2 December 2023. doi: [10.2210/pdb8BLT/pdb](doi.org/10.2210/pdb8BLT/pdb)

Lodola A, Branduardi D, De Vivo M, Capoferri L, Mor M, Piomelli D, Cavalli A. 2012. A catalytic mechanism for cysteine N-terminal nucleophile hydrolases, as revealed by free energy simulations. PLoS ONE 7:e32397. doi: [10.1371/journal.pone.0032397](doi.org/10.1371/journal.pone.0032397)

Mingrone G, Panunzi S, De Gaetano A, Guidone C, Iaconelli A, Leccesi L, Nanni G, Pomp A, Castagneto M, Ghirlanda G, Rubino F. 2012. Bariatric surgery versus conventional medical therapy for type 2 diabetes. N Engl J Med 366:1577–1585. doi: [10.1056/NEJMoa1200111](doi.org/10.1056/NEJMoa1200111)

Mingrone G, Panunzi S, De Gaetano A, Guidone C, Iaconelli A, Nanni G, Castagneto M, Bornstein S, Rubino F. 2015. Bariatric–metabolic surgery versus conventional medical treatment in obese patients with type 2 diabetes: 5 year follow-up of an open-label, single-centre, randomised controlled trial. Lancet 386:964–973. doi: [10.1016/S0140-6736(15)00075-6](doi.org/10.1016/S0140-6736(15)00075-6)

Moore RE, Townsend SD. 2019. Temporal development of the infant gut microbiome. Open Biol 9:190128. doi: [10.1098/rsob.190128](doi.org/10.1098/rsob.190128)

Mueller NT, Whyatt R, Hoepner L, Oberfield S, Dominguez-Bello MG, Widen EM, Hassoun A, Perera F, Rundle A. 2015a. Prenatal exposure to antibiotics, cesarean section and risk of childhood obesity. Int J Obes 39:665–670. doi: [10.1038/ijo.2014.180](doi.org/10.1038/ijo.2014.180)

Mueller NT, Bakacs E, Combellick J, Grigoryan Z, Dominguez-Bello MG. 2015b. The infant microbiome development: Mom matters. Trends Mol Med 21:109–117. doi: [10.1016/j.molmed.2014.12.002](doi.org/10.1016/j.molmed.2014.12.002)

Nair PP, Gordon M, Reback J. 1967. The enzymatic cleavage of the carbon-nitrogen bond in 3&alpha;, 7&alpha;, 12&alpha;-trihydroxy-5-&beta;-cholan-24-oylglycine. J Biol Chem 242:7–11. doi: [10.1016/S0021-9258(18)96311-8](doi.org/10.1016/S0021-9258(18)96311-8)

Nothias L-F, Petras D, Schmid R, Dührkop K, Rainer J, Sarvepalli A, Protsyuk I, Ernst M, Tsugawa H, Fleischauer M, Aicheler F, Aksenov AA, Alka O, Allard P-M, Barsch A, Cachet X, Caraballo-Rodriguez AM, Da Silva RR, Dang T, Garg N, Gauglitz JM, Gurevich A, Isaac G, Jarmusch AK, Kameník Z, Kang KB, Kessler N, Koester I, Korf A, Le Gouellec A, Ludwig M, Martin H. C, McCall L-I, McSayles J, Meyer SW, Mohimani H, Morsy M, Moyne O, Neumann S, Neuweger H, Nguyen NH, Nothias-Esposito M, Paolini J, Phelan VV, Pluskal T, Quinn RA, Rogers S, Shrestha B, Tripathi A, van der Hooft JJJ, Vargas F, Weldon KC, Witting M, Yang H, Zhang Z, Zubeil F, Kohlbacher O, Böcker S, Alexandrov T, Bandeira N, Wang M, Dorrestein PC. 2020. Feature-based molecular networking in the GNPS analysis environment. Nat Methods 17:905–908. doi: [10.1038/s41592-020-0933-6](doi.org/10.1038/s41592-020-0933-6)

Quinn RA, Melnik AV, Vrbanac A, Fu T, Patras KA, Christy MP, Bodai Z, Belda-Ferre P, Tripathi A, Chung LK, Downes M, Welch RD, Quinn M, Humphrey G, Panitchpakdi M, Weldon KC, Aksenov A, da Silva R, Avila-Pacheco J, Clish C, Bae S, Mallick H, Franzosa EA, Lloyd-Price J, Bussell R, Thron T, Nelson AT, Wang M, Leszczynski E, Vargas F, Gauglitz JM, Meehan MJ, Gentry E, Arthur TD, Komor AC, Poulsen O, Boland BS, Chang JT, Sandborn WJ, Lim M, Garg N, Lumeng JC, Xavier RJ, Kazmierczak BI, Jain R, Egan M, Rhee KE, Ferguson D, Raffatellu M, Vlamakis H, Haddad GG, Siegel D, Huttenhower C, Mazmanian SK, Evans RM, Nizet V, Knight R, Dorrestein PC. 2020. Global chemical effects of the microbiome include new bile-acid conjugations. Nature 579:123–129. doi: [10.1038/s41586-020-2047-9](doi.org/10.1038/s41586-020-2047-9)

Sehgal K, Khanna S. 2021. Gut microbiome and <i>Clostridioides difficile</i> infection: a closer look at the microscopic interface. Therap Adv Gastroenterol 14:1756284821994736. doi: [10.1177/1756284821994736](doi.org/10.1177/1756284821994736)

Shalon D, Culver RN, Grembi JA, Folz J, Treit PV, Shi H, Rosenberger FA, Dethlefsen L, Meng X, Yaffe E, Aranda-Díaz A, Geyer PE, Mueller-Reif JB, Spencer S, Patterson AD, Triadafilopoulos G, Holmes SP, Mann M, Fiehn O, Relman DA, Huang KC. 2023. Profiling the human intestinal environment under physiological conditions. Nature 617:581–591. doi: [10.1038/s41586-023-05989-7](doi.org/10.1038/s41586-023-05989-7)

Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, Ideker T. 2003. Cytoscape: A software environment for integrated models of biomolecular interaction networks. Genome Res 13:2498–2504. doi: [10.1101/gr.1239303](doi.org/10.1101/gr.1239303)

Song Z, Cai Y, Lao X, Wang X, Lin X, Cui Y, Kalavagunta PK, Liao J, Jin L, Shang J, Li J. 2019. Taxonomic profiling and populational patterns of bacterial bile salt hydrolase (BSH) genes based on worldwide human gut microbiome. Microbiome 7:9. doi: [10.1186/s40168-019-0628-3](doi.org/10.1186/s40168-019-0628-3)

Stellwag EJ, Hylemon PB. 1976. Purification and characterization of bile salt hydrolase from <i>Bacteroides fragilis</i> subsp. <i>fragilis</i>. BBA - Enzymol 452:165–176. doi: [10.1016/0005-2744(76)90068-1](doi.org/10.1016/0005-2744(76)90068-1)

Stewart CJ, Ajami NJ, O’Brien JL, Hutchinson DS, Smith DP, Wong MC, Ross MC, Lloyd RE, Doddapaneni HV, Metcalf GA, Muzny D, Gibbs RA, Vatanen T, Huttenhower C, Xavier RJ, Rewers M, Hagopian W, Toppari J, Ziegler AG, She JX, Akolkar B, Lernmark A, Hyoty H, Vehik K, Krischer JP, Petrosino JF. 2018. Temporal development of the gut microbiome in early childhood from the TEDDY study. Nature 562:583–588. doi: [10.1038/s41586-018-0617-x](doi.org/10.1038/s41586-018-0617-x)

Turnbaugh PJ, Ley RE, Hamady M, Fraser-Liggett CM, Knight R, Gordon JI. 2007. The human microbiome project. Nature 449:804–810. doi: [10.1038/nature06244](doi.org/10.1038/nature06244)

Wang M, Carver JJ, Phelan VV, Sanchez LM, Garg N, Peng Y, Nguyen DD, Watrous J, Kapono CA, Luzzatto-Knaan T, Porto C, Bouslimani A, Melnik AV, Meehan MJ, Liu WT, Crüsemann M, Boudreau PD, Esquenazi E, Sandoval-Calderón M, Kersten RD, Pace LA, Quinn RA, Duncan KR, Hsu CC, Floros DJ, Gavilan RG, Kleigrewe K, Northen T, Dutton RJ, Parrot D, Carlson EE, Aigle B, Michelsen CF, Jelsbak L, Sohlenkamp C, Pevzner P, Edlund A, McLean J, Piel J, Murphy BT, Gerwick L, Liaw CC, Yang YL, Humpf HU, Maansson M, Keyzers RA, Sims AC, Johnson AR, Sidebottom AM, Sedio BE, Klitgaard A, Larson CB, Boya CAP, Torres-Mendoza D, Gonzalez DJ, Silva DB, Marques LM, Demarque DP, Pociute E, O’Neill EC, Briand E, Helfrich EJN, Granatosky EA, Glukhov E, Ryffel F, Houson H, Mohimani H, Kharbush JJ, Zeng Y, Vorholt JA, Kurita KL, Charusanti P, McPhail KL, Nielsen KF, Vuong L, Elfeki M, Traxler MF, Engene N, Koyama N, Vining OB, Baric R, Silva RR, Mascuch SJ, Tomasi S, Jenkins S, Macherla V, Hoffman T, Agarwal V, Williams PG, Dai J, Neupane R, Gurr J, Rodríguez AMC, Lamsa A, Zhang C, Dorrestein K, Duggan BM, Almaliti J, Allard PM, Phapale P, Nothias LF, Alexandrov T, Litaudon M, Wolfender JL, Kyle JE, Metz TO, Peryea T, Nguyen DT, VanLeer D, Shinn P, Jadhav A, Müller R, Waters KM, Shi W, Liu X, Zhang L, Knight R, Jensen PR, Palsson B, Pogliano K, Linington RG, Gutiérrez M, Lopes NP, Gerwick WH, Moore BS, Dorrestein PC, Bandeira N. 2016. Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking. Nat Biotechnol 34:828–837. doi: [10.1038/nbt.3597](doi.org/10.1038/nbt.3597)

Yang I, Corwin EJ, Brennan PA, Jordan S, Murphy JR, Dunlop A. 2016. The infant microbiome: implications for infant health and neurocognitive development. Nurs Res 65:76–88. doi: [10.1097/NNR.0000000000000133](doi.org/10.1097/NNR.0000000000000133)

Zapata RC, Zhang D, Chaudry B, Osborn O. 2021. Self-Administration of drugs in mouse models of feeding and obesity. JoVE 62775. doi: [10.3791/62775](doi.org/10.3791/62775)
